pageserver/src/http/routes.rs

```diff
@@ -453,9 +461,15 @@ async fn tenant_load_handler(request: Request<Body>) -> Result<Response<Body>, A
     let ctx = RequestContext::new(TaskKind::MgmtRequest, DownloadBehavior::Warn);
 
     let state = get_state(&request);
-    mgr::load_tenant(state.conf, tenant_id, state.remote_storage.clone(), &ctx)
-        .instrument(info_span!("load", tenant = %tenant_id))
-        .await?;
+    mgr::load_tenant(
+        state.conf,
+        tenant_id,
+        state.broker_client,
+        state.remote_storage.clone(),
```

Comment:
Weird cloning a `&'static BrokerClient`, not wrong but I think references are `Copy`?
```suggestion
        state.remote_storage,
```